lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -625,6 +625,15 @@ data ApiPoolId
     | ApiPoolId PoolId
     deriving (Eq, Generic, Show)
 
+instance FromText AccountPublicKey where
```

Comment:
Why is this instance located here :thinking: ? Or perhaps, if `AccountPublicKey` is an API type, what about renaming it to follow the same convention as other API types (either starting with `Api..` or, wrapped in `ApiT`).

Reply:
changed `AccountPublicKey` in `Api.Types` to `ApiAccountPublicKey` and propagated through code base in https://github.com/input-output-hk/cardano-wallet/pull/1421/commits/ddec074ddadbddb70b0cfbd8899efa145b9c0dc7